File: app/src/main/java/com/starostinvlad/fan/Adapters/SlidePagerAdapter.java
package com.starostinvlad.fan.Adapters;

import java.util.ArrayList;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentStatePagerAdapter;

public class SlidePagerAdapter extends FragmentStatePagerAdapter {

    public SlidePagerAdapter(@NonNull FragmentManager fm, int behavior, ArrayList<Fragment> fragmentArrayList) {
        super(fm, behavior);
        this.fragmentArrayList = fragmentArrayList;
    }

    private ArrayList<Fragment> fragmentArrayList;

    @NonNull
    @Override
    public Fragment getItem(int position) {
        return fragmentArrayList.get(position);
    }

    @Override
    public int getCount() {
        return fragmentArrayList.size();
    }
}

File: app/src/main/java/com/starostinvlad/fan/Adapters/SlidePagerAdapter.kt
package com.starostinvlad.fan.Adapters

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentStatePagerAdapter
import java.util.*
import kotlin.Throws

class SlidePagerAdapter(fm: FragmentManager, behavior: Int, private val fragmentArrayList: ArrayList<Fragment>) : FragmentStatePagerAdapter(fm, behavior) {
    override fun getItem(position: Int): Fragment {
        return fragmentArrayList[position]
    }

    override fun getCount(): Int {
        return fragmentArrayList.size
    }
}